pkg/api/norman/customization/cluster/actionhandler.go

```diff
@@ -2,7 +2,6 @@ package cluster
 
 import (
 	"fmt"
-
```

Comment:
nit: don't remove line.

Reply:
Yep! Caught that yesterday when I received golint errors